Logging tests can fail at the wait-for-namenodes container

The logging tests assert that there are log events from all containers.

For the wait-for-namenodes container, the logs from the following command are logged:

```
/stackable/hadoop/bin/hdfs haadmin -getServiceState $namenode_id
```

This command creates the following logs as long as a namenode is not ready yet:

```
<log4j:event logger="org.apache.hadoop.ipc.Client" timestamp="1703075527534" level="INFO" thread="main">
<log4j:message><![CDATA[Retrying connect to server: test-hdfs-custom-log-namenode-default-0.test-hdfs-custom-log-namenode-default.kuttl-test-square-labrador.svc.cluster.local/10.244.1.51:8020
. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=1, sleepTime=1000 MILLISECONDS)]]></log4j:message>
</log4j:event>
```

But if all namenodes are ready then apparently nothing is logged. This causes the test to fail:

```
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation | running command: [sh -c kubectl exec --namespace=$NAMESPACE hdfs-test-runner-0 -- python /tmp/test_log_aggregation.py -n $NAMESPACE]
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation | Traceback (most recent call last):
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation |   File "/tmp/test_log_aggregation.py", line 47, in <module>
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation |     check_sent_events()
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation |   File "/tmp/test_log_aggregation.py", line 42, in check_sent_events
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation |     sentEvents['sentEventsTotal'] > 0, \
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation | AssertionError: No events were sent in "filteredCustomLogConfigDataNode0WaitForNameNodes".
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation | command terminated with exit code 1
logging_hadoop-3.2.4_zookeeper-latest-3.8.3/6-test-log-aggregation | test step failed 6-test-log-aggregation
```

--- a/tests/templates/kuttl/logging/03-install-hdfs-automatic-log.yaml.j2
+++ b/tests/templates/kuttl/logging/03-install-hdfs-automatic-log.yaml.j2
@@ -86,10 +86,13 @@ spec:
                 console:
                   level: INFO
                 file:
-                  level: INFO
+                  # Set the log level to DEBUG because if the NameNodes
+                  # are already ready then nothing is logged at the INFO
+                  # level.
+                  level: DEBUG
                 loggers:
                   ROOT:
-                    level: INFO
+                    level: DEBUG
   journalNodes:
     roleGroups:
       default:

--- a/tests/templates/kuttl/logging/04-install-hdfs-custom-log.yaml.j2
+++ b/tests/templates/kuttl/logging/04-install-hdfs-custom-log.yaml.j2
@@ -49,7 +49,9 @@ metadata:
   name: hdfs-wait-for-namenodes-log-config
 data:
   wait-for-namenodes.log4j.properties: |
-    log4j.rootLogger=INFO, FILE
+    # Set the log level to DEBUG because if the NameNodes are already
+    # ready then nothing is logged at the INFO level.
+    log4j.rootLogger=DEBUG, FILE
     log4j.appender.FILE=org.apache.log4j.FileAppender
     log4j.appender.FILE.File=/stackable/log/wait-for-namenodes/wait-for-namenodes.log4j.xml
     log4j.appender.FILE.layout=org.apache.log4j.xml.XMLLayout

